ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -445,14 +445,16 @@ function osd_activate {
   fi
   mkdir -p /var/lib/ceph/osd
   chown ceph. /var/lib/ceph/osd
+  # resolve /dev/disk/by-* names
+  ACTUAL_OSD_DEVICE=$(readlink -f ${OSD_DEVICE})
```

Comment:
Yeah I had something like that originally, but readlink does nothing if it's not a link and just prints /dev/sdf2 for example.